app/src/main/java/com/example/mvp/ui/signin/SignInActivity.kt

```diff
@@ -0,0 +1,66 @@
+package com.example.mvp.ui.signin
+
+import android.content.Intent
+import android.view.View
+
+import com.example.mvp.R
+import com.example.mvp.data.source.local.AppDatabase
+import com.example.mvp.data.source.local.dao.UserDaoImpl
+import com.example.mvp.ui.base.BaseActivity
+import com.example.mvp.ui.main.MainActivity
+import com.example.mvp.ui.signup.SignUpActivity
+import com.example.mvp.utils.showToast
+import kotlinx.android.synthetic.main.activiity_signin.*
+
+class SignInActivity : BaseActivity(), SignInContract.View, View.OnClickListener {
+
+    private var mAppDatabase : AppDatabase? = null
+    private var mSignInPresenter: SignInPresenter? = null
+    private var mUserDaoImpl: UserDaoImpl? = null
```

Comment:
Ở Kotlin thì không cần thêm m, i, ... vì nó tự sinh ra java 
```suggestion
    private var appDatabase : AppDatabase? = null
    private var signInPresenter: SignInPresenter? = null
    private var userDaoImpl: UserDaoImpl? = null
```